doc/usage/installation.rst

```diff
@@ -31,7 +31,8 @@ Install either ``python3-sphinx`` using :command:`apt-get`:
 
 ::
 
-   $ apt-get install python3-sphinx
+   $ apt-get install python3-pip
```

Comment:
The above text also mentions the `python3-sphinx` package. It should be updated too.

Additionally, it would be better to mention not to mix Sphinx package from PyPI and distro's. Usually, people get confused about the difference between them.